tests/testthat/test-json_nest.R

```diff
@@ -5,3 +5,21 @@ test_that("`json_nest()` works", {
     nested
   })
 })
+
+test_that("`json_nest()` works remotely", {
+  # FIXME: add "mssql" when ready
+  skip_if_src_not("postgres")
+  con <- my_test_src()$con
+
+  withr::defer(
+    try(dbExecute(con, "DROP TABLE iris"))
+  )
+
+  dbWriteTable(con, "iris", iris)
+  iris_remote <- tbl(con, "iris")
+
+  expect_snapshot({
```

Comment:
The snapshot test is likely to fail for other databases. We can make it exclusive to Postgres, or try to test an invariant.

Reply:
It was exclusive to Postgres due to the `skip_if_src_not("postgres")` but we now also use the `variant` argument so we can have multiple snapshots when we support other databases. https://github.com/cynkra/dm/pull/975/commits/7c030ebc5cb1e910ddc3f2ca234713888fa0cab4